### DISCUSSION TITLE
Infiltration: Instead of a fake player, consider assigning "captures" to a team member instead

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#677] [Feature Requests]
Infiltration uses regular captures by fake "blue"/"green" players to represent the timer of the game. Instead of a dummy player, a real player from the team could be used, e.g. one with the most kills in that timeframe.

Good:
- simplifies code
- player can be rewarded for doing good things
 
Bad:
 - might confuse players

**Comment by feikname** [comment]
I'm in favor of doing this.

**Comment by NotAFile** [comment]
we could help lower the confusion with the new center-screen messages

**Comment by feikname** [comment]
does BetterSpades adopt those?

**Comment by xtreme8000** [comment]
Not yet, I still need to decide how to display all the different message types (which are all basicly the same?).

**Comment by NotAFile** [comment]
well, they just fall back to chat messages, so I don't see anything wrong with just sending them.
